src/styles/cards.scss

```diff
@@ -0,0 +1,95 @@
+@import "./variables.scss";
```

Comment:
add variables import to the main file at the first line and remove it from there